src/instructlab/client.py

```diff
@@ -32,7 +32,7 @@ def list_models(
         client = OpenAI(
             base_url=api_base,
             api_key=api_key,
-            timeout=DEFAULT_CONNECTION_TIMEOUT,
+            timeout=httpx.Timeout(30.0),
```

Comment:
Why not use `DEFAULT_CONNECTION_TIMEOUT` here anymore?

Reply:
No good reason, updated